### PR TITLE
Add protobuf license in LICENSE and longabout.html

### PR DESCRIPTION
**Pull request by mpirvu**
Fixes: #8733

Signed-off-by: Marius Pirvu <mpirvu@ca.ibm.com>

### CODE REVIEW

**Comment by fjeremic** [comment]
I thought we are removing protobufs as a dependency in #8486. Do we need this license change?

**Comment by mpirvu** [comment]
@pshipton  'longabout.html' file does not have a copyright message and I am not sure it should have one.

**Comment by mpirvu** [comment]
We are removing protobuf, but until that change is done it's safer to add the license text.

**Comment by pshipton** [comment]
> We are removing protobuf, but until that change is done it's safer to add the license text.

Agreed. The builds containing protobuf should declare the license.

File: longabout.html [hunk 1]
```diff
@@ -259,5 +259,45 @@ <h4>
...
+</h4>
+<p>
+Copyright 2008 Google Inc.  All rights reserved.
+</p>
+<p>
+Redistribution and use in source and binary forms, with or without
+modification, are permitted provided that the following conditions are
+met:
+
+    * Redistributions of source code must retain the above copyright
```
Comment (pshipton) [inline longabout.html]: Some html is required here to fix the formatting. As it stands, it shows as one paragraph.
```
Redistribution and use in source and binary forms, with or without modification, are permitted provided that the following conditions are met: * Redistributions of source code must retain the above copyright notice, this list of conditions and the following disclaimer. * Redistributions in binary form must reproduce the above copyright notice, this list of conditions and the following disclaimer in the documentation and/or other materials provided with the distribution. * Neither the name of Google Inc. nor the names of its contributors may be used to endorse or promote products derived from this software without specific prior written permission.
```

Comment (mpirvu) [reply longabout.html]: Corrected.

**Comment by pshipton** [comment]
lgtm. I think we don't this in the 0.19.0 release since protobuf (jitserver) isn't enabled for Java 14.

**Pull request merged**